### PR TITLE
toggle 'open' class on click

### PR DESCRIPTION
**Pull request by LibbyChapman**
@incuna/frontend please review

Added functionality to make elements toggle an 'open' class on click
(Added new file to hold miscellaneous style guide JS)

### CODE REVIEW

File: scripts/style-guide.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+(function ($) {
+    'use strict';
+
+    $(document).ready(function () {
+        // Add a class 'toggleItem' to elements to have them toggle a class of 'open' when clicked (to demonstrate show/hide effects etc)
+        $(document).on('click', '.toggleItem', function (e) {
```
Comment (wytrych) [inline scripts/style-guide.js]: Should this be `.toggle-item` class instead?

Comment (LibbyChapman) [inline scripts/style-guide.js]: oh yes, I will fix. Thanks.

**Comment by hippogriffic** [comment]
nice 👍

**Comment by ninjanails** [comment]
💃  awesome

**Comment by LibbyChapman** [comment]
thank you :)

**Comment by henrahmagix** [comment]
👍

**Pull request merged**